app/build.gradle

```diff
@@ -275,6 +285,7 @@ dependencies {
     alphaImplementation "com.google.firebase:firebase-messaging-ktx:$firebaseMessagingVersion"
     betaImplementation "com.google.firebase:firebase-messaging-ktx:$firebaseMessagingVersion"
     greatfireImplementation "com.google.firebase:firebase-messaging-ktx:$firebaseMessagingVersion"
+    fdroidImplementation "com.google.firebase:firebase-messaging-ktx:$firebaseMessagingVersion"
```

Comment:
I don't know if this ends up in the final APK, but `com.google.mlkit` and `com.google.firebase` are both proprietary libs, so can't be included in f-droid.org.

Reply:
it looks like mlkit is being used to auto-detect languages when editing descriptions.

Reply:
firebase messaging also seems to be used for some kind of notifications if users are logged in.

Reply:
in the dependency list here i see both mlkit and firebase but only mlkit is flagged: https://gitlab.com/fdroid/fdroiddata/-/merge_requests/12056  in the "Scan APK" list i only see https://pushnotifications-73c5e.firebaseio.com though

Reply:
it looks like the solution in the original wikipedia build.gradle was not to add these dependencies for the fdroid flavor and to include stubbed out versions of the classes that relied on them.  i'll do the same here.